### PR TITLE
Create FB ad landing page

### PR DESCRIPTION
**Pull request by sandymak**
#### What’s this PR do?
- Create new landing page for FB ad to send users here to sign up & get link to download app texted to them

#### How was this tested? How should this be reviewed?
- invoked locally

#### What are the relevant tickets?
- https://trello.com/c/rGc2kXNR/328-create-new-landing-page-for-fb-ads-to-send-a-text-to-download-the-app

#### Questions / Considerations
~~❌ Missing copy for mobile text flow & need to add that to mobile Commons~~
❌ Need to have copy checked prior to launching

### CODE REVIEW

File: api/services/partnersData.js [hunk 1]
```diff
@@ -244,6 +244,21 @@ const partnersData = {
     signUpButtonText: 'Get Started',
     campaignKey: 'OP836EE83472E6391BE0A81A0DEAF35F24',
   },
+  'get-app': {
+    name: 'Meet your new self-care sidekick.',
+    imageUrl: '/images/partners/shine-app.jpg',
+    copy:
+      "<p style='font-size: 18px'>1) Wake up daily to Shine Texts: bite-sized, research-backed advice.</p> <p style='font-size: 18px'>2) Listen to Shine Talks on topics like self-love, anxiety, and burnout.</p> <p style='font-size: 18px'>3) Remind yourself what you're grateful for every day.</p> <hr> <p style='font-weight: 800;'>93% of people felt happier and more confident after making Shine part of their daily routine.</p> <p style='font-size: 18px'>Enter your info below to get a text to download our app:</p>",
```
Comment (kaladin9017) [inline api/services/partnersData.js]: Could you wrap this in a `<div style='font-size: 18px'></div>` ?

File: api/services/partnersData.js [hunk 2]
```diff
@@ -244,6 +244,21 @@ const partnersData = {
...
+    confirmation: {
+      imageUrl: '/images/confirmation-header-happy-dance.gif',
+      copy: "You're all signed up!",
+    },
+    campaignKey: 'OPDB9D9EC3F2E3A13D022DF83A30987B2B',
```
Comment (kaladin9017) [inline api/services/partnersData.js]: We usually make this into an environment variable. `process.env.FB_AD_MOBILECOMMONS_KEY `. But it looks like sometimes we don't do this 🤷‍♂️

Comment (jonuy) [reply api/services/partnersData.js]: Oh yea, I told Sandy that we probably don't need to do this anymore. The values we've put in there generally don't change at all, and they don't really need to be treated as secrets either.

Comment (kaladin9017) [reply api/services/partnersData.js]: Copy Copy.

**Pull request merged**